File: src/BreakfastProvider.Api/Models/Responses/RecipeReviewResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public class RecipeReviewResponse
{
    public string ReviewId { get; set; } = string.Empty;
    public string RecipeName { get; set; } = string.Empty;
    public string ReviewerName { get; set; } = string.Empty;
    public int Rating { get; set; }
    public string Comments { get; set; } = string.Empty;
    public List<string> Tags { get; set; } = [];
    public DateTime CreatedAt { get; set; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/RecipeReviewResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;
import java.util.List;

/** Twin of C# {@code RecipeReviewResponse}. */
public record RecipeReviewResponse(
        String reviewId,
        String recipeName,
        String reviewerName,
        int rating,
        String comments,
        List<String> tags,
        Instant createdAt) {
}
